L1Trigger/L1TTrackMatch/python/L1TrackJetProducer_cfi.py

```diff
@@ -1,61 +1,77 @@
 import FWCore.ParameterSet.Config as cms
 from L1Trigger.VertexFinder.VertexProducer_cff import VertexProducer
 
-
+#prompt jet selection
 L1TrackJets = cms.EDProducer('L1TrackJetProducer',
 	L1TrackInputTag= cms.InputTag("TTTracksFromTrackletEmulation", "Level1TTTracks"),
-	L1PVertexCollection = cms.InputTag("VertexProducer", VertexProducer.l1VertexCollectionName.value()),
-	MaxDzTrackPV = cms.double( 0.5 ),
+#        L1TrackInputTag= cms.InputTag("TTTracksFromTrackletEmulation","Level1TTTracks","IN"),
```

Comment:
Is it still needed (eg to revert back later)? If not please remove

Reply:
Yes that also i missed to remove from the period i ran the comparison. (F/W was compatible with old cmssw files, so it was the only way we could run.)